src/index.js

```diff
@@ -0,0 +1 @@
+export * from './rules/javascript';
```

Comment:
oh, that should never have been submitted. No, it isn't and it was part of me experimenting around with a real eslint-plugin. I'll get rid of it.